README.md

```diff
@@ -1,20 +1,25 @@
-# Notation
+# notation-go
 
 [![Build Status](https://github.com/notaryproject/notation-go/actions/workflows/build.yml/badge.svg?event=push&branch=main)](https://github.com/notaryproject/notation-go/actions/workflows/build.yml?query=workflow%3Abuild+event%3Apush+branch%3Amain)
-[![codecov](https://codecov.io/gh/notaryproject/notation-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-go)
+[![Codecov](https://codecov.io/gh/notaryproject/notation-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/notaryproject/notation-go.svg)](https://pkg.go.dev/github.com/notaryproject/notation-go@main)
 
-A collection of libraries for supporting Notation sign, verify, push, pull of oci artifacts. Based on Notary Project standard.
+notation-go offers libraries for signing, verifying, uploading, and downloading OCI artifacts using [Notary Project specifications](https://github.com/notaryproject/specifications). notation-go is used by [notation](https://github.com/notaryproject/notation) CLI for signing and verifying artifacts.
```

Comment:
At another look, I think we should not use the term `OCI artifacts` because it's not a defined term in the OCI spec. In addition, notation-go only upload/download signatures, it doesn't do the same to the target artifact.
```suggestion
notation-go offers libraries for signing, verifying, uploading, and downloading signatures following the [Notary Project specifications](https://github.com/notaryproject/specifications). notation-go is used by [notation](https://github.com/notaryproject/notation) CLI for signing and verifying artifacts.
```